src/main/java/com/synopsys/integration/jenkins/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -46,6 +46,7 @@ public class SecurityScanStep extends Step implements Serializable {
     private String bridge_blackduck_scan_failure_severities;
 //    private Boolean bridge_blackduck_automation_fixpr;
     private Boolean bridge_blackduck_automation_prcomment;
+    private String BRIDGE_BLACKDUCK_DOWNLOAD_URL;
```

Comment:
As per @sig-tithi 's last comment,she said let's not change all parameters to uppercase. 
So need to check whether we should make this lowercase like other parameters.

Reply:
Done